datastore/models.py

```diff
@@ -21,6 +22,15 @@
 from collections import defaultdict, OrderedDict
 import itertools
 
+# This is perhaps unneccessary...callers of `run_meter` could just
+# pass the class name directly. For now, though, the existing code
+# depends on being about to pass 'residential' as a meter type.
+#
+# Also: this could live in a separate settings file
+METER_CLASS_CHOICES = {
+    'residential': DefaultResidentialMeter,
```

Comment:
Is there any reason not to have this be this, which might be more straightforward?

        'DefaultResidentialMeter': DefaultResidentialMeter,